### DISCUSSION TITLE
Which version of data-diff you have forked?

### DISCUSSION THREAD

**Discussion by Ryuk-me** [erezsh/reladiff#14] [General]
I believe you have forked an older version of data diff tool. Could you please tell me the reason for doing so?

Thank you.

**Comment by erezsh** [comment]
Hello @Ryuk-me ,

Do you mean why I didn't fork the most recent version?

I forked the last version that I know was stable. I then went manually through each commit that came after, and only applied it to reladiff if I thought it was good and necessary.

The commits that I didn't include are almost entirely either dbt-related, or are internal refactors. I also omitted the sqlserver implementation, because I don't think that it's correct.

But if there's anything important in data-diff that I missed adding to reladiff, I'm open to including it.

**Reply by grahamwetzler** [reply]
Just curious, why did you choose to not include the dbt stuff?

**Reply by erezsh** [reply]
Hi @grahamwetzler !

The dbt integration was a feature for reading the run config automatically from dbt, instead of having to specify it. 

I removed the dbt integration because I thought it was a bad design. I might consider re-adding it as a separate command, e.g. reladiff-dbt. It would probably be best as an entirely separate tool that imports reladiff, since it doesn't need any special API or access to reladiff internals (iirc).

Either way, you can currently use Reladiff from dbt without any issue, either as a Python library or as a shell command.

**Reply by mmoyer-pax** [reply]
Hey @erezsh, thanks for maintaining this tool. Has there been any further consideration on including a dbt integration in some form (either directly here as a separate command or as a separate tool that imports reladiff)? 

data-diff's dbt integration _really_ simplified running diffs (to the point where I can have my not-so-technical analysts easily running diffs when they make a dbt model change). We need to upgrade our dbt version from v1.7 to v1.8, and at first glance it seems like the last released version of data-diff may be incompatible with dbt v1.8, so having streamlined dbt diffing in this tool would be awesome.

**Reply by erezsh** [reply]
I'm all for adding dbt integration as a separate tool/cmd, but I don't have the dbt expertise, or frankly the time, to do it myself.

If anyone wants to take it on, I'll be more than happy to help in any way I can, such as provide advice, make necessary changes to reladiff, etc.

**Comment by Ryuk-me** [comment]
Hey @erezsh thanks for replying.
I'll surely let you know if there is anything which needs to be included.
Great work !!
